modules/framePreProc/framePreProc.py

```diff
@@ -0,0 +1,48 @@
+
+THRESHHOLD = 30
+
+class FramePreProc:
+
+    """
+
+        Filters out useless telemetry data i.e data whose change in euler angles from the previous state exceeds 30 degrees
+
+        Attributes
+        ----------
+        eulerDict : dict
+            dictionary holding the eulerAnglesOfCamera data from telemetry
+        eulerDictLast : dict
+            dictionary holding the previous eulerAnglesOfCamera to compare to
+
+    """
+
+    def __init__(self, eulerDict, eulerDictLast):
+
+        self.eulerDict = eulerDict
+        self.eulerDictLast = eulerDictLast
+
+    def filter(self):
+
+        """
+
+        Finds the difference between present euler angles and those of the previous state.
+
+        Returns
+        -------
+        Boolean: 'True' if absolute difference per angle does not exceed 30 degrees else returns 'False'
+
+        """
+
```

Comment:
Add a check here, if self.lastEulerDict is None, then just return None as we can't perform processing.